<?xml version="1.0"?>
<testsuites>
    <testsuite name="UnitTests.MainClassTest" tests="3" time="0.289" failures="1" errors="0" skipped="0">
        <testcase classname="UnitTests.MainClassTest" name="TestPropertyValue" time="0.146"/>
        <testcase classname="UnitTests.MainClassTest" name="TestMethodUpdateValue" time="0.001"/>
        <testcase classname="UnitTests.MainClassTest" name="TestFailure" time="0.092">
            <failure>
                MESSAGE:
                Expected failure
                Expected: 30
                But was: 20

                +++++++++++++++++++
                STACK TRACE:
                at UnitTests.MainClassTest.TestFailure () [0x00000]
                at &lt;0x00000&gt; &lt;unknown method&gt;
                at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (object,object[])
                at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr,
                System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture)
                [0x00000]
            </failure>
        </testcase>
    </testsuite>
</testsuites>
